dev/jenkins/build.sh

```diff
@@ -65,7 +65,7 @@ export MAVEN_OPTS="-Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simple
 
 if [ -z "${ALLUXIO_BUILD_FORKCOUNT}" ]
 then
-  ALLUXIO_BUILD_FORKCOUNT=4
+  ALLUXIO_BUILD_FORKCOUNT=1
```

Comment:
do you need to set this env var to 4 in the github action yamls so they run with the same settings as before this change? i'm not sure how many builds run through this codepath without setting this env var; would need some time to comb through them all